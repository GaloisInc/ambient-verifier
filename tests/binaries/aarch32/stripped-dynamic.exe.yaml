statechart:
  name: IHaveAchievedNothing
  root state:
    name: Root
    initial: NormalExecution
    states:
      - name: NormalExecution
        description: |
          Execution before the Weird Machine starts
        transitions:
          - target: WMEntered
            event: EnterWeirdMachine(0x544)
      - name: WMEntered
        description: |
          The Weird Machine has been entered
        type: final
