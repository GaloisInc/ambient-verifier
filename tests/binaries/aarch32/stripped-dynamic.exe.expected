successful: 0
failed: 0
entryPointAddr: 0x55c
